Dockerfile

```diff
@@ -129,6 +133,5 @@ COPY start /app/bin/
 RUN chmod a+rx /app/bin/start
 
 EXPOSE 8080 4443
-USER 1000
-ENTRYPOINT ["/app/bin/start"]
-CMD ["runserver"]
+USER weblate
+CMD ["/app/bin/start", "runserver"]
```

Comment:
Can you please explain reasons for this change?

Reply:
When image fails to start, not setting a app-specific entrypoint makes it easier to `docker run image sh` and inspect things.

Changing this is not absolute necessary. If it can cause problems I will revert it.

Reply:
I just don't want to break existing workflows. AFAIR this is used since the container exists.

Reply:
:ok: reverted this one too

Reply:
Thanks. We can land that separately, but I'm just worried of the outcome of such change as it might have impact on how people use the container (the docker run would be changed from running weblate by default with provided args to run anything, so the first passed arg which was previously arg for weblate would be now name of program to be executed).